Source/CocoaMQTT.swift

```diff
@@ -207,8 +207,20 @@ public class CocoaMQTT: NSObject, CocoaMQTTClient, CocoaMQTTDeliverProtocol {
     /// Enable auto-reconnect mechanism
     public var autoReconnect = false
     
-    /// Auto reconnect time interval
-    public var autoReconnectTimeInterval: UInt16 = 20
+    /// Reconnect time interval
+    ///
+    /// - note: This value will be increased with `autoReconnectTimeInterval *= 2`
+    ///         if reconnect failed
+    public var autoReconnectTimeInterval: UInt16 = 1 // starts from 1 second
```

Comment:
@HJianBo Ideally, would like to have a separate class for reconnect functionality. 
Which shall have 
init() -> which initializes the array of timeouts
reconnectIfRequired() -> which shall check whats the next timer and starts the timer. If we have reached the maxReconnectTime, we reset
reset() -> reset the properties like 
                    self.reconnectExponetials = []
                    self.reconectCheckTimes = 0
                    self.autoReconnTimer = nil

Reply:
Yes, it a good idea to improve readability and usability of code. I'll fix later  : ]